virtcontainers/qemu.go

```diff
@@ -90,7 +90,7 @@ const (
 	qmpCapErrMsg                      = "Failed to negoatiate QMP capabilities"
 	qmpCapMigrationBypassSharedMemory = "bypass-shared-memory"
 	qmpExecCatCmd                     = "exec:cat"
-	qmpMigrationWaitTimeout           = 5 * time.Second
+	qmpMigrationWaitTimeout           = 10 * time.Second
```

Comment:
Same comment as above - why not make this value arm64-specific?

Reply:
@jodh-intel @bergwolf thanks, I agree with that. I will put all the value to arch base file. offering usingbypassSharedMemory cap is not necessary for other arch, so I will offer a const value as a  switch to decide if  ExecSetMigrationCaps should be executed.